make/run

```diff
@@ -32,11 +32,13 @@ kubectl get storageclass persistent 2>/dev/null || {
     kubectl create -f -
 }
 
-source env/defaults.env
+DOMAIN=${DOMAIN:-$(grep '^DOMAIN=' env/defaults.env | sed 's/DOMAIN=//')}
```

Comment:
Proposal: Go with `cd ${GIT_ROOT}` at the beginning, and remove all other references